### ISSUE TITLE
UX of setting rlimits

### ISSUE THREAD

**Issue by igalic** [iocage/libiocage#361]
currently it's possible to set rlimits, but it has no effect on a default setup

```
rctl: RACCT/RCTL present, but disabled; enable using kern.racct.enable=1 tunable
```

We should check for this `sysctl`, and either set it ourselves, or recommend our users set it themselves.

*****

i don't think we can set this from ioc:

```
sysctl: oid 'kern.racct.enable' is a read only tunable
sysctl: Tunable values are set in /boot/loader.conf
```

so what's left is to recommending to the users they do this themselves.

**Comment by urosgruber** [comment]
I'm ok to just tell the user it's disabled and maybe some help message.

**Comment by gronke** [comment]
There is another occasion where host features could be disabled: Secure-VNET requires ipfw to be enabled with specific features. In both cases it makes sense to do the compatibility checks early on, so that we do not produce jail zombies.

Would you find it useful to include the commands to enable such host features in the message or is the plain warning about missing features enough to figure it out?

**Comment by urosgruber** [comment]
@gronke commands would be great.

**Comment by gronke** [comment]
Implemented and documented in the Wiki https://github.com/iocage/libiocage/wiki/Resource-Limits

**Comment by urosgruber** [comment]
@gronke are you sure resource-limits are implemented the way wiki is describing?

**Comment by gronke** [comment]
> @gronke are you sure resource-limits are implemented the way wiki is describing?

@urosgruber Can you please make an example of what is not working as expected?

**Comment by urosgruber** [comment]
So if I call `ioc set vcpu=vmem:deny=128M/jail foobar` I get an error **The config property 'vcpu' of jail 'foobar' is unknown.**

But `ioc set vmemoryuse=128M foobar` works.
